### ISSUE TITLE
Server logo customization in home page list

### ISSUE THREAD

**Issue by Doncellup** [pterodactyl/panel#4139]
### Is there an existing feature request for this?

- [X] I have searched the existing issues before opening this feature request.

### Describe the feature you would like to see.

Hello, First of all congratulations for your wonderful software! It is truly amazing!

After several months of use, I had an idea to improve this one :

It would be nice to be able to customize the server logo displayed in the list of servers on the home page:
![image](https://user-images.githubusercontent.com/90469321/173245070-fe661484-ee8b-4bd5-95af-a99f0becc12a.png)

### Describe the solution you'd like.

For this, we could have a category in the settings tab of the servers. We could in this one put the class of the icon FontAwesome, like this :
![image](https://user-images.githubusercontent.com/90469321/173245338-634bb0f1-6457-4cfa-8bed-9860502ce745.png)

### Additional context to this request.

Thanks in advance,
Good day to you !

**Comment by FoksVHox** [comment]
Why? What benefit would this have?

**Comment by Doncellup** [comment]
> Why? What benefit would this have?

An advantage in terms of customization.  And when you have a lot of servers it's much easier to find your way around with logos

**Comment by FoksVHox** [comment]
In that case wouldn't this be sort of a duplicate of #498

**Comment by camwhi-te** [comment]
Not really, it seems to be a request to implement more customization over practicality. It'd make more sense for the egg to be assigned an image instead of per-server basis, as that'd lead to a lot more data being stored in the database

**Comment by Doncellup** [comment]
> Not really, it seems to be a request to implement more customization over practicality. It'd make more sense for the egg to be assigned an image instead of per-server basis, as that'd lead to a lot more data being stored in the database

Exactly, in my opinion it's still better to be able to choose one icon per server, as in my initial suggestion. It just adds an additional data in the database (besides it's a short data, for example "fas fa-gamepad").

**Comment by DaneEveritt** [comment]
I wouldn't be adding arbitrary font-awesome icon support (only the ones we use are explicitly included, and I'm moving new UIs away from them). If anything it would make more sense for Eggs to specify an image that is used.

**Comment by anondev-sudo** [comment]
It would be nice if server icon could be set per egg, so you can easily recognise for example discord bot hosting servers, game servers, ...

**Comment by itzzjustmateo** [comment]
How it looks it got approved, but it isn't there D:

**Comment by FoksVHox** [comment]
> How it looks it got approved, but it isn't there D:

@itzzjustmateo Apprently yeah, but just because it got approved, it doesn't mean it's at the top of the priority list, but it'll come when it comes.

**Comment by itzzjustmateo** [comment]
ahh okay, I thought like Approved is there for was added xDD but still a great idea with this logo thingy I think 😄
